suite-native/module-add-accounts/src/hooks/useAddCoinAccount.ts

```diff
@@ -281,38 +294,46 @@ export const useAddCoinAccount = () => {
             return;
         }
 
-        const selectedType = accountType ?? getDefaultAccountType({ network });
+        const selectedType = accountType ?? getDefaultAccountType({ networkSymbol });
 
         const canAddAccount = checkCanAddAccount({
-            network,
+            networkSymbol,
             accountType: selectedType,
         });
 
         if (!canAddAccount) {
             return;
         }
 
+        const network = getNetworkToAdd({ networkSymbol, accountType: selectedType });
+
         const accountsWithTypeLength = getAccountsForNetworSymbolkWithAccountType({
-            network,
+            networkSymbol,
             accountType: selectedType,
         }).length;
 
         navigateToSuccessorScreen({
             flowType,
-            networkSymbol: network.symbol,
+            networkSymbol,
             accountType: selectedType,
             accountIndex: accountsWithTypeLength,
         });
 
         const account = await dispatch(
             addAndDiscoverNetworkAccountThunk({
                 network,
-                accountType: selectedType,
                 deviceState: device.state,
             }),
         ).unwrap();
 
         if (!account) {
+            let screen = 'accounts';
```

Comment:
`accounts` is not a valid route from `routes.ts`. Why do you assign it here?

Reply:
good catch, that is a mistake

Reply:
9dd5a668c4